README.md

```diff
@@ -51,9 +51,9 @@ Check out the [Getting Started Guide][getting_started] to create your first Kube
 
 This provider's versions are able to install and manage the following versions of Kubernetes:
 
-| Kubernetes Version          | v1.19 | v1.20 | v1.21 | v1.22 | v1.23 |
-| --------------------------- | ----- | ----- | ----- | ----- | ----- |
-| CloudStack Provider  (v0.4) |   ✓   |   ✓   |   ✓   |   ✓   |   ✓   |
+| Kubernetes Version          | v1.22 | v1.23 |
```

Comment:
Are older versions not supported anymore with v0.4.x?

Reply:
They are, but since the prebuilt images are not available, have removed them from the docs